xml/templates/dao.tpl

```diff
@@ -30,7 +30,7 @@ class {$table.className} extends CRM_Core_DAO {ldelim}
 {foreach from=$table.fields item=field}
     /**
 {if $field.comment}
-     * {$field.comment}
+     * {$field.comment|regex_replace:"/\n[ ]*/":"\n* "}
```

Comment:
@seamuslee001 this looks like the comment change - but where are the trailing commas coming from?